webapp/package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "history": "^4.6.3",
     "marked": "^0.3.6",
-    "moment": "^2.24.0",
+    "moment": "^2.8.1",
```

Comment:
Please revert the change. The moment was already in higher version

Reply:
Done :)